RELEASES.md

```diff
@@ -1,6 +1,31 @@
 Unreleased
 ---------
 
+* [make `calc` require the sequence of relation/proof-s to have the same indentation](https://github.com/leanprover/lean4/pull/1844),
+  and [add `calc` alternative syntax allowing underscores `_` in the first relation](https://github.com/leanprover/lean4/pull/1844)
+
+  the flexible indentation in `calc` was often used to align relations/proofs
+  ```lean
+  example (x y : Nat) : (x + y) * (x + y) = x * x + y * x + x * y + y * y :=
+    calc
+        (x + y) * (x + y) = (x + y) * x + (x + y) * y       := by rw [Nat.mul_add]
+                        -- improper indentation
+                        _ = x * x + y * x + (x + y) * y     := by rw [Nat.add_mul]
+                        _ = x * x + y * x + (x * y + y * y) := by rw [Nat.add_mul]
+                        _ = x * x + y * x + x * y + y * y   := by rw [←Nat.add_assoc]
+  ```
+
+  this is no longer legal, but the alternative syntax accomplishes essentially the same goal with proper indentation:
```

Comment:
```suggestion
  This is no longer legal.  The new syntax puts the first term right after the `calc` and each step has the same indentation:
```

Reply:
The first term does not need to be right after the `calc`, at least not in the same line. For instance:

```
calc
  <expr>_0 
  '_' 'op_1' <expr>_1 ':=' <proof>_1
  '_' 'op_2' <expr>_2 ':=' <proof>_2
  ...
  '_' 'op_n' <expr>_n ':=' <proof>_n
```

Reply:
Yeah ideally we would have a formatter standardizing the whole thing. I would prefer having the expression right after the `calc` but if it's a long expression it might make sense to newline it.

In any case I'm fine with the change you propose, it's just that it sounds like it is mandatory